client/my-sites/google-my-business/index.js

```diff
@@ -31,6 +32,18 @@ const loadKeyringAndSiteSettingsMiddleware = ( context, next ) => {
 	] ).then( next );
 };
 
+const redirectUnauthorized = ( context, next ) => {
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+	const siteIsGMBEligible = isSiteGoogleMyBusinessEligible( state, siteId );
+	const canUserManageOptions = canCurrentUser( state, siteId, 'manage_options' );
+	if ( ! siteIsGMBEligible || ! canUserManageOptions ) {
+		context.redirect( `/stats/${ context.params.site }` );
```

Comment:
is there a reason to use `context` instead of `page` here? In testing I saw that `context` would sometimes prevent the redirect rendering, where using `page` would briefly show the GMB page before redirecting.

Reply:
I think `context.redirect` should be use in place of `page.redirect` in most case. It's just  a shortend for it on the client but it's also isomorphic and will do an HTTP redirect if run on the server